### PR TITLE
GEN-1529 / Add Perils to QuickAddinfoDialog

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Implement QuickAddinfoDialog
- Add Perils and adjust layout

https://github.com/HedvigInsurance/racoon/assets/6661511/a8523c59-6e0a-45bd-89d2-ac7f29dd963e

Next PR: Design tweaks (Perils are unevenly spaced it looks like)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3740** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3740?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3741** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3741?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3741?utm_source=stack-comment).

File: apps/store/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -72,13 +72,13 @@ export const HeaderWithTrigger = ({ children }: HeaderWithTriggerProps) => {
 const OpenIcon = styled(PlusIcon)({
   flexShrink: 0,
   display: 'block',
-  '[data-state=open] &': { display: 'none' },
+  '[data-state=open] > &': { display: 'none' },
```
Comment (gustaveen) [inline apps/store/src/components/Accordion/Accordion.tsx:75]: Make sure Accordion icons in an open Dialog are styled correctly

File: apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx [hunk 2]
```diff
@@ -18,21 +18,25 @@ export const QuickAddInfoDialog = ({ children, Header }: Props) => {
...
+              <DialogBody>{children}</DialogBody>
+            </Space>
+          </ScrollWrapper>
         </Wrapper>
       </DialogContent>
     </Dialog.Root>
   )
 }
 
+const DIALOG_HEIGHT = 'min(45rem, calc(100vh - 2rem))'
```
Comment (robinandeer) [inline apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx:38]: nit: do we want to use theme spacing here?

File: apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx [hunk 3]
```diff
@@ -42,11 +46,23 @@ const DialogContent = styled(Dialog.Content)({
 const Wrapper = styled.div({
   position: 'relative',
   width: 'min(35.5rem, calc(100vw - 2rem))',
```
Comment (robinandeer) [inline apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx:48]: nit: theme?

File: apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx [hunk 4]
```diff
@@ -58,7 +74,15 @@ const DialogHeader = styled.div({
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'center',
-  paddingTop: theme.space.xl,
+  paddingTop: '4.5rem',
```
Comment (robinandeer) [inline apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx:77]: theme?

**Review by robinandeer — APPROVED** [review]
Added via Giphy
looks great

File: apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx (hunk 4 above)
Comment (gustaveen) [reply apps/store/src/components/QuickAdd/QuickAddInfoDialog.tsx:77]: We don't have that one in the theme, it's more of a edge case :)

**Pull request merged**